hangman.rb

```diff
@@ -0,0 +1,31 @@
+random_words = ["camera", "photo", "picture", "landscape", "light", "shadow", "lens", "portrait"]
+#the number is one more
+random_num = rand(0..7).to_i
+chosen_word = random_words[random_num]
```

Comment:
Since Ruby is actually really magical... there's actually code to do exactly what you just described! I'll let you dig into it on your own. My suggestion would be to google `ruby array documentation` and look into the different methods you can do with arrays.